### PR TITLE
Test for whether ES index was backed up the previous day to S3

### PR DESCRIPTION
**Pull request by annebyrne**

### CODE REVIEW

File: app/com/gu/contentapi/sanity/ElasticSearchSnapshotTest.scala [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import java.time.LocalDate
+
+@ProdOnly
+class ElasticSearchSnapshotTest(context: Context) extends SanityTestBase(context) {
+
+    "Elasticsearch backup via snapshot" should "have been updated yesterday on S3" in {
+
+      val yesterdayDate = LocalDate.now().minusDays(1L)
+      val objectKey = s"PROD-ZEBRA/meta-${yesterdayDate}-00-00.dat"
+      val s3Client: AmazonS3Client = new AmazonS3Client()
```
Comment (LATaylor-guardian) [inline app/com/gu/contentapi/sanity/ElasticSearchSnapshotTest.scala]: The client should be a member of the class and therefore only created once when the class is instantiated as opposed to each time the test is run.

File: app/com/gu/contentapi/sanity/ElasticSearchSnapshotTest.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import com.amazonaws.services.s3.AmazonS3Client
+import java.time.LocalDate
+
+@ProdOnly
+class ElasticSearchSnapshotTest(context: Context) extends SanityTestBase(context) {
+
+    "Elasticsearch backup via snapshot" should "have been updated yesterday on S3" in {
+
+      val yesterdayDate = LocalDate.now().minusDays(1L)
+      val objectKey = s"PROD-ZEBRA/meta-${yesterdayDate}-00-00.dat"
```
Comment (LATaylor-guardian) [inline app/com/gu/contentapi/sanity/ElasticSearchSnapshotTest.scala]: `PROD-ZEBRA` will change to `PROD-AARDVARK` when we switch stacks (which we do when we upgrade Elasticsearch) meaning this test will break.

Comment (LATaylor-guardian) [inline app/com/gu/contentapi/sanity/ElasticSearchSnapshotTest.scala]: I'm just looking to see how to do this so it do doesn't break, or maybe we don't care and we fix when we switch stacks.

**Comment by LATaylor-guardian** [comment]
👍 (when build passes)

**Comment by mchv** [comment]
@annebyrne @LATaylor-guardian 
Don't we need a `S3Policy` to allow the instance to access `S3` [like here] ?(https://github.com/guardian/content-api/blob/master/concierge/cloudformation.yaml#L109-L121)

**Comment by mchv** [comment]
Sorry missed the previous `PR`!

**Pull request merged**